NewForce-at-Mountwest/bangazon-site-the-blue-ferrets
# Ps ticket13 - User Can View Product Details and Add the product to their Order

# Description
Tickt #13 - Users can view Product Details

Given user is viewing any page that contains a hyperlink for a product
When user performs a gesture on the product hyperlink
Then user will be shows the product detail page containing title, description, quantity available, price/unit, and a button labeled Add to Order

Given 1+ users have purchased a product
When a user views the details
Then the quantity display should show how many are remaining, not how many were originally for sale

Fixes # (issue)

New feature.

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions

Go to Home on the Navigation Bar, Select any of the Product Hyperlinks, you will see the Details of that Product including title, description, quantity available, price/unit, and a button labeled Add to Order (If you aren't logged in, you won't see this button, but you will see a message to Login to Add this product to your Order).  If you click the button, the item will be added to your Order (Cart) and then you will be routed back to the Index view of Products where the Quantity Available of the product you added will be 1 less.


# Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
